wake/loaders/productListingPage.ts

```diff
@@ -107,9 +108,10 @@ export interface Props {
   pageHref?: string;
 
   /**
+   * @title Partner Param
    * @description page param to partners page
    */
-  slug?: RequestURLParam;
+  partnerAlias?: RequestURLParam;
```

Comment:
It brokes the current sites using wake api, right?

Reply:
we may have other clients that may need it.

Can you keep the slug property, tag it as deprecated, and define parterAlias with slug if it is not defined (i.e. `partnerAlias ??= slug`)?